server/monitor.go

```diff
@@ -1106,6 +1106,8 @@ type RemoteLeafOptsVarz struct {
 	LocalAccount string   `json:"local_account,omitempty"`
 	TLSTimeout   float64  `json:"tls_timeout,omitempty"`
 	URLs         []string `json:"urls,omitempty"`
+	DenyExports  []string `json:"deny_exports,omitempty"`
```

Comment:
Should we say Deny property that has two sub properties, Exports and Imports?
Otherwise LGTM.

Reply:
wrapped them in a dedicated struct